packages/veritone-react-common/src/components/DataTable/PaginatedTableFooter.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import cx from 'classnames';
+import IconButton from 'material-ui/IconButton';
+import FirstPageIcon from 'material-ui-icons/FirstPage';
+import KeyboardArrowLeft from 'material-ui-icons/KeyboardArrowLeft';
+import KeyboardArrowRight from 'material-ui-icons/KeyboardArrowRight';
+import LastPageIcon from 'material-ui-icons/LastPage';
+import { TableCell } from 'material-ui/Table';
+import Select from 'material-ui/Select';
+import { MenuItem } from 'material-ui/Menu';
+
+import { func, number, shape, objectOf, string, any } from 'prop-types';
+import RefreshButton from '/components/RefreshButton';
+import styles from './styles/index.scss';
+
+export default class PaginatedTableFooter extends React.Component {
+  static propTypes = {
+    page: number.isRequired,
+    perPage: number.isRequired,
+    onChangePerPage: func.isRequired,
+    onRefreshPageData: func,
+    rowCount: number,
+    colSpan: number
+  };
+
+  static defaultProps = {
+    colSpan: 1
+  };
+
+  handleFirstPageButtonClick = event => {
+    this.props.onChangePage(event, 0);
+  };
+
+  handleBackButtonClick = event => {
+    this.props.onChangePage(event, this.props.page - 1);
+  };
+
+  handleNextButtonClick = event => {
+    this.props.onChangePage(event, this.props.page + 1);
+  };
+
+  handleLastPageButtonClick = event => {
+    this.props.onChangePage(
+      event,
+      Math.max(0, Math.ceil(this.props.rowCount / this.props.perPage) - 1),
+    );
+  };
+
+  render() {
+    const { rowCount, page, perPage } = this.props;
+    const firstItem = page * perPage + 1;
+    const lastItem = Math.min(
+      page * perPage + perPage,
+      rowCount
+    );
+
+    return (
+      <TableCell
+        className={styles['paginated-footer']}
+        colSpan={this.props.colSpan}        
+        style={{
+          textAlign: 'right'
+          // display: 'flex',
```

Comment:
clean up if we're not planning to use these

Reply:
done